perf/istio-install/setup_istio.sh

```diff
@@ -33,6 +33,33 @@ function setup_admin_binding() {
     --user=$(gcloud config get-value core/account) || true
 }
 
+function install_prometheus() {
+  local DIRNAME="$1" # should be like tools/perf/istio/tmp
+  local PROMETHEUS_INSTALL="${DIRNAME}/../../prometheus-install"
+
+  # Create GCP SSD Storage class for Prometheus to use. May not work outside GKE
+  kubectl apply -f "${PROMETHEUS_INSTALL}/ssd-storage-class.yaml"
+
+  helm fetch stable/prometheus-operator --untar --untardir "${PROMETHEUS_INSTALL}"
+
+  # Store original context namespace so it can be reset at the end
+  local ORIG_CTX=$(kubectl config current-context)
+  local ORIG_NS=$(kubectl config get-contexts ${ORIG_CTX} --no-headers | tr -s ' ' | cut -d ' ' -f 5)
+  # Prometheus operator chart doesn't respect --namespace, all objects are
+  # deployed to the default namespace.
+  kubectl config set-context $(kubectl config current-context) --namespace=istio-system
+  helm template "${PROMETHEUS_INSTALL}/prometheus-operator/"\
+    -f "${PROMETHEUS_INSTALL}/prometheus-operator-values.yaml"\
+    --set-file .Values.prometheus.prometheusSpec.additionalScrapeConfigs="${PROMETHEUS_INSTALL}/prometheus-scrape-configs.yaml"\
+    | kubectl apply -f -
```

Comment:
`kubectl apply -n istio-system` should work without all the context hacks?

Reply:
I tried that, but had difficulty... it was almost a month ago now, and I can't recall what the problem was.  I will test it out and report back.